### ISSUE TITLE
Specify that focus neighbor "not set" means an empty NodePath, not null

### ISSUE THREAD

**Issue by ShatReal** [godotengine/godot-docs#7730]
**Your Godot version:**
4.1.1

**Issue description:**
I tried to set focus neighbor property on a Control node to null but it gave me an invalid set index error. This is because I need to have an empty nodepath (^"") not null.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-property-focus-neighbor-bottom

**Comment by AThousandShips** [comment]
The documentation doesn't instruct to use `null` but that it's unset, which means the default value which is shown to be `^""`

**Comment by ShatReal** [comment]
my smol brain didn't know what unset meant

**Comment by tetrapod00** [comment]
Yeah, the status quo for this seems *fine*. But I don't think changing this to explicitly mention an empty nodepath is wrong either.
